### PR TITLE
Added documentation examples for `consider-using-any-or-all`.

### PR DESCRIPTION
**Pull request by mbyrnepr2**
`bad.py` examples taken directly from the functional tests.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|    | :bug: Bug fix          |
|    | :sparkles: New feature |
|    | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Refs https://github.com/PyCQA/pylint/issues/5953

### CODE REVIEW

File: doc/data/messages/c/consider-using-any-or-all/good.py [hunk 1]
```diff
@@ -1 +1,4 @@
-# This is a placeholder for correct code for this message.
+items = (1, 2, 3, 4, 5)
+
+any(item % 2 == 0 for item in items)
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/c/consider-using-any-or-all/good.py:3]: Could you keep the functions from the 'bad' example please ?

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Thank you for adding this !

File: doc/data/messages/c/consider-using-any-or-all/good.py (hunk 1 above)
Comment (mbyrnepr2) [reply doc/data/messages/c/consider-using-any-or-all/good.py:3]: Do you mean like this?
`any_even = any(item % 2 == 0 for item in items)`?

File: doc/data/messages/c/consider-using-any-or-all/good.py [hunk 2]
```diff
@@ -1 +1,4 @@
-# This is a placeholder for correct code for this message.
+items = (1, 2, 3, 4, 5)
+
+any_even = any(item % 2 == 0 for item in items)
+all_even = all(item % 2 == 0 for item in items)
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/c/consider-using-any-or-all/good.py:4]:
```suggestion
def any_even(items):
    """Return True if the list contains any even numbers"""
    return any(item % 2 == 0 for item in items)

def all_even(items):
    """Return True if the list contains all even numbers"""
    return all(item % 2 == 0 for item in items)
```

File: doc/data/messages/c/consider-using-any-or-all/good.py [hunk 3]
```diff
@@ -1 +1,9 @@
-# This is a placeholder for correct code for this message.
+items = (1, 2, 3, 4, 5)
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/c/consider-using-any-or-all/good.py:1]:
```suggestion
```

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**